test/BindingTests.cs

```diff
@@ -100,5 +103,114 @@ public BindingTests(ITestOutputHelper output)
                 await host.StopAsync();
             }
         }
+
+        [Fact]
+        public async Task BindToBlobViaParameterName()
+        {
+            using (JobHost host = TestHelpers.GetJobHost(this.loggerFactory, nameof(this.ActivityTriggerAsNumber)))
+            {
+                await host.StartAsync();
+
+                string connectionString = AmbientConnectionStringProvider.Instance.GetConnectionString(ConnectionStringNames.Storage);
```

Comment:
FYI, we could simplify this test to avoid Blobs and be totally in-memory. 
We have a short binding extension (you could copy from https://github.com/Azure/azure-webjobs-sdk/pull/1513  ) that operates entirely in-memory.

Reply:
Good to know. One thing I like about end-to-end tests, though, is that they sometimes make for good mini-samples.